### PR TITLE
added required field and removed novalidate from form

### PR DESCRIPTION
**Pull request by Vidyaa123**
Signed-off-by: Vidyaa <mailvidyaa@gmail.com>

**Description**

This PR fixes # 298

**Notes for Reviewers**
Used required attribute in the input fields of Newsletter form to not proceed without all the required inputs and removed "novalidate" from the form to allow validation.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Comment by YashKamboj** [comment]
LGTM!! 👍👌

**Review by YashKamboj — APPROVED** [review]

**Comment by YashKamboj** [comment]
@warunicorn19 Views??

File: docs/_includes/subscribe.html [hunk 1]
```diff
@@ -11,27 +11,23 @@ <h3 class="section-subheading text-muted">{{ site.data.sitetext.subscribe.text |
...
             
-          <input type="text" value="" placeholder="First Name" name="FNAME" class="form-control" id="mce-FNAME" >
+          <input type="text" value="" placeholder="First Name" name="FNAME" class="form-control" id="mce-FNAME" required />
           <p class="help-block text-danger"></p>
-            <input type="text" value="" placeholder="Last Name" name="LNAME" class="form-control" id="mce-LNAME" >
+            <input type="text" value="" placeholder="Last Name" name="LNAME" class="form-control" id="mce-LNAME" required />
           <p class="help-block text-danger"></p>
           <input type="email" value="" name="EMAIL" class="form-control" id="mce-EMAIL" placeholder="Email Address" 
-          required style="white-space: nowrap;display: inline-block;">
+          required style="white-space: nowrap;display: inline-block;" required/>
```
Comment (warunicorn19) [inline docs/_includes/subscribe.html:21]: this `input` already has a required attribute.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: docs/_includes/subscribe.html (hunk 1 above)
Comment (Vidyaa123) [reply docs/_includes/subscribe.html]: removed as requested. and pushed the changes

**Review by warunicorn19 — COMMENTED** [review]
looks good, thank you @Vidyaa123

**Review by warunicorn19 — APPROVED** [review]

**Pull request merged**